wrappers/PhyML/phyml/phyml_wrapper.xml

```diff
@@ -0,0 +1,374 @@
+<?xml version="1.0"?>
+<tool id="phyml_wrapper" name="PhyML" version="1.0">
+        <description> Estimates maximum likelihood phylogenies from alignments of nucleotide or amino-acid sequences </description>
+
+        <requirements>
+            <requirement type="package" version="3.1">phyml</requirement>
+        </requirements>
+
+        <command>
+	<![CDATA[
+                filename=./`basename $seq_file_name` &&
+                
+		cp $seq_file_name \$filename &&
+                
+		phyml -i \$filename
+
+		-d "$sequences.data_type"
+		#if $sequences.data_type == "nt"
+			-m "$sequences.selection.model_name"
+			#if $sequences.selection.model_name == "HKY85"
+				#if $sequences.selection.ratio.ts_tv_ratio == "e"
+					-t "$sequences.selection.ratio.ts_tv_ratio"
+				#else 
+					-t "$sequences.selection.ratio.fixed"
+				#end if
+			#end if
+			#if $sequences.selection.model_name == "K80"
+				#if $sequences.selection.ratio.ts_tv_ratio == "e"
+					-t "$sequences.selection.ratio.ts_tv_ratio"
+				#else
+					-t "$sequences.selection.ratio.fixed"
+				#end if
+			#end if
+			#if $sequences.selection.model_name == "F84"
+				#if $sequences.selection.ratio.ts_tv_ratio == "e"
+					-t "$sequences.selection.ratio.ts_tv_ratio"
+				#else
+					-t "$sequences.selection.ratio.fixed"
+				#end if
+			#end if
+		#end if	
+		#if $sequences.data_type == "nt"
+			-f "$sequences.eq_freq"
+		#end if	
+		#if $sequences.data_type == "aa"
```

Comment:
Double check this and next condition. If this is correct behavior, then merge two ifs.